### PR TITLE
Fix handling of Dynamic Field: Type

### LINKED ISSUE #104: Level1 Dynamic Field Type Issue
Dynamic Level1 Streaming, trying use the field "Type" to identify trades vs updates. When I add "Type" to my list of fields it causes all the other data (Bid,Ask,Last etc) to come back weird (1s, and 2s,).

Pic1: Type commented out -- everything works just fine 
image

Pic 2: Type included -- everything comes back weird
image

### PR DESCRIPTION
**Pull request by BunkerCoder**
fixes #104

### CODE REVIEW

**Comment by BunkerCoder** [comment]
Realize that I made a fundamental mistake, and Type needs to return P (summary) or Q (update), not just Q. Need to refactor to make this solution work.
@mathpaquette  - Will you pull if I get this right, or should we wait for #107?

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs [hunk 1]
```diff
@@ -295,7 +295,7 @@ public static Level1DynamicFields Parse(string message, DynamicFieldset[] fields
             int tick = default;
             int tickID = default;
             int totalVolume = default;
-            string type = default;
+            string type = "Q";
```
Comment (mathpaquette) [inline src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs:298]: so this will only be Q update ?

**Review by mathpaquette — APPROVED** [review]

File: src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs (hunk 1 above)
Comment (BunkerCoder) [reply src/IQFeed.CSharpApiClient/Streaming/Level1/Level1DynamicFields.cs:298]: I goofed.  It should also return P.  See comments above. this does not work properly.

**Comment by mathpaquette** [comment]
> Realize that I made a fundamental mistake, and Type needs to return P (summary) or Q (update), not just Q. Need to refactor to make this solution work.
> @mathpaquette - Will you pull if I get this right, or should we wait for #107?

Ah you just closed it?? I think we should get the fix ASAP and merge. Will still need to support until next major release.

**Comment by BunkerCoder** [comment]
Yes - Just closed now.  It doesn't break, it just doesn't return Type=P when the message is a summary message.

**Comment by mathpaquette** [comment]
> Yes - Just closed now. It doesn't break, it just doesn't return Type=P when the message is a summary message.
I can wait your other commit. no stress.

**Comment by BunkerCoder** [comment]
Didn't need to refactor - just fix parsing.

**Comment by mathpaquette** [comment]
> Didn't need to refactor - just fix parsing.

Are you confident now :) hehe

**Comment by BunkerCoder** [comment]
Hard to be confident after stupid brain flatulence like that, but, Yes...

**Review by mathpaquette — APPROVED** [review]

**Pull request merged**